### PR TITLE
Document Select2's dynamic option creation

### PR DESCRIPTION
**Pull request by meysammahfouzi**

### CODE REVIEW

File: modules/system/assets/ui/docs/select.md [hunk 1]
```diff
@@ -43,6 +43,15 @@ Add the `select-no-search` CSS class to disable searching.
         </select>
     </div>
 
+### Dynamic option cretion
```
Comment (mjauvin) [inline modules/system/assets/ui/docs/select.md:46]:
```suggestion
### Dynamic option creation
```

File: modules/system/assets/ui/docs/select.md [hunk 2]
```diff
@@ -43,6 +43,15 @@ Add the `select-no-search` CSS class to disable searching.
         </select>
     </div>
 
+### Dynamic option creation
+
+In addition to a prepopulated menu of options, Select2 can dynamically create new options from text input by the user in the search box. This feature is called "tagging". To enable tagging, set the `tags` option to `true`:
+
+    <select
+        class="form-control custom-select"
+        data-tags=true
```
Comment (bennothommo) [inline modules/system/assets/ui/docs/select.md:52]: @meysammahfouzi 
```suggestion
        data-tags="true"
```

**Review by bennothommo — DISMISSED** [review]

File: modules/system/assets/ui/docs/select.md (hunk 2 above)
Comment (meysammahfouzi) [reply modules/system/assets/ui/docs/select.md:52]: Is this necessary? I test `data-tags=true` and it worked.

Comment (bennothommo) [reply modules/system/assets/ui/docs/select.md:52]: @meysammahfouzi Our standards are to have quotes around attribute values.

**Pull request merged**